File: output/qwen_2.5_72b_instruct/avatar/direct_translation/python/java/failed_translation/atcoder_ABC119_B.java
import java.util.Scanner;

public class atcoder_ABC119_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        double[] x = new double[N];
        for (int i = 0; i < N; i++) {
            String input = scanner.next();
            input = input.replace("JPY", "* 1.0").replace("BTC", "* 380000.0");
            x[i] = eval(input);
        }
        double sum = 0;
        for (double value : x) {
            sum += value;
        }
        System.out.println(sum);
    }

    public static double eval(String expression) {
        return new javax.script.ScriptEngineManager().getEngineByName("JavaScript").eval(expression);
    }
}

File: output/llama_4_maverick/avatar/algo_based_translation/java/python/successful_translation/atcoder_ABC119_B.py

def main():
    N = int(input())
    x = [0] * N
    u = [0] * N
    for i in range(N):
        x[i], u[i] = input().split()
        x[i] = float(x[i])
    total_sum = sum(x[i] * (380000 if u[i] == "BTC" else 1) for i in range(N))
    print(total_sum)

if __name__ == "__main__":
    main()

